gate-core/src/main/groovy/com/netflix/spinnaker/gate/model/manageddelivery/DeliveryConfig.java

```diff
@@ -22,8 +22,10 @@
 
 @Data
 public class DeliveryConfig {
+  String apiVersion;
```

Comment:
`apiVersion` is nullable, and not required right now. I forget - does this allow it? or do you need a default value of null?

Reply:
Added a test & it seems fine